frontend/src/pages/ProgressionChecker/ProgressionChecker.tsx

```diff
@@ -257,6 +258,7 @@ const ProgressionChecker = () => {
                     <GridView
                       uoc={structure[group].content[subgroup].UOC}
                       subgroupTitle={subgroup}
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
can we remove this elsint disable line (same goes at line ~250)?  Not sure why this was added in the git diff?

Reply:
yep should be done